django/forms/utils.py

```diff
@@ -94,8 +94,8 @@ class ErrorDict(UserDict, RenderableErrorMixin):
     template_name_text = 'django/forms/errors/dict/text.txt'
     template_name_ul = 'django/forms/errors/dict/ul.html'
 
-    def __init__(self, data=None, renderer=None):
-        super().__init__(data)
+    def __init__(self, inititems=(), *, renderer=None, **kwargs):
+        super().__init__(inititems, **kwargs)
```

Comment:
I think this is _probably_ fine/correct enough, but it is subtly different in the exception message thrown when bad data is given to it. I don't know that it _matters_, though? Someone more certain than I should make a decision on that.

```
In [1]: class T1(dict):
   ...:     def __init__(self, a=(), *, x=None, **kwargs):
   ...:         super().__init__(a, **kwargs)
   ...:
In [2]: class T2(dict):
   ...:     def __init__(self, *args, x=None, **kwargs):
   ...:         super().__init__(*args, **kwargs)
   ...:
   ...:
In [3]: T1((), ())
TypeError: __init__() takes from 1 to 2 positional arguments but 3 were given
In [4]: T2((), ())
TypeError: dict expected at most 1 argument, got 2
In [5]: dict((), ())
TypeError: dict expected at most 1 argument, got 2
In [6]: dict.__new__
Out[6]: <function dict.__new__(*args, **kwargs)>
```
and it also precludes passing in a kwarg of `inititems` (here `a` just because I'm apparently being confusing) _in addition_ to a positional iterable:
```
In [7]: T1({'test': 1}, a=1)
TypeError: __init__() got multiple values for argument 'a'
In [8]: T2({'test': 1}, a=1)
Out[8]: {'test': 1, 'a': 1}
In [9]: dict({'test': 1}, a=1)
Out[9]: {'test': 1, 'a': 1}
```
which again ... I don't know that it _matters_.

Reply:
Maybe I was supposed to use `/` here instead of `*`? I'm trying to make the first argument optional/positional only, let me try that one more time :)

Reply:
I think `def __init__(self, inititems=(), /, *, renderer=None, **kwargs):` is the correct spelling.

This allows for the following:

```
>>> ErrorDict({'foo': ['bar']}, inititems=['ok'])
{'foo': ['bar'], 'inititems': ['ok']}
```

When two positional arguments are given the error is slightly different from a normal dict, but I think it's more correct:

```
>>> ErrorDict({'foo': ['bar']}, {'bar': ['baz']})
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __init__() takes from 1 to 2 positional arguments but 3 were given
```

Reply:
Although, your suggested `def __init__(self, *args, renderer=None, **kwargs):` is more pleasing to the eye and basically achieves the same thing. I'm changing my mind and will just adopt that signature.